scripts/jrelease.java

```diff
@@ -101,38 +101,33 @@ public static void main(String... args) throws Exception {
                 + nextReleaseVersion
                 + "\"/}' -i .github/ISSUE_TEMPLATE/bug_report.yml");
 
-        // 6. Send a message to Gitter channel `Starting release process for vx.y.z`
-        printStep(6, "Send a message to Gitter channel `Starting release process for v" + nextReleaseVersion + "`");
-        print("Press enter to continue");
-        System.in.read();
```

Comment:
Umm, why have we deleted `Send a message to Gitter channel `Starting release process` step?

Reply:
I couldn't find it anymore in the knowledge base doc, so I thought it wasn't really important

Reply:
I changed the knowledgebase back to what it was (plain and clear checklist) before the intermediate steps were added.
I did this, because the idea was that the script was already acting as a reminder for the smaller steps.
Whatever you consider is something that needs to be triggered should be either automated or reminded.

Reply:
@sunix WDYT? Any follow-up?

Reply:
I don't have strong opinion on this: we are already sending a message on gitter once the release is done. @rohanKanojia what do you think?

Reply:
In my opinion, it's good practice to share it with the community that we've initiated the release process. Since we're already doing it, let's continue doing it,